src/material-experimental/mdc-slider/slider.e2e.spec.ts

```diff
@@ -6,11 +6,133 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-/* tslint:disable-next-line:no-unused-variable */
-import {MatSlider} from './index';
-
-// TODO(wagnermaciel): Implement this in a separate PR
+import {clickElementAtPoint, getElement, Point} from '@angular/cdk/testing/private/e2e';
+import {Thumb} from '@material/slider';
+import {browser, by, element, ElementFinder} from 'protractor';
 
 describe('MDC-based MatSlider' , () => {
-  it('does nothing yet', async () => {});
+  const getStandardSlider = () => element(by.id('standard-slider'));
+  const getDisabledSlider = () => element(by.id('disabled-slider'));
+  const getRangeSlider = () => element(by.id('range-slider'));
+
+  beforeEach(async () => await browser.get('mdc-slider'));
+
+  describe('standard slider', async () => {
+    let slider: ElementFinder;
+    beforeAll(() => { slider = getStandardSlider(); });
+
+    it('should update the value on click', async () => {
+      await setValueByClick(slider, 15);
+      expect(await getSliderValue(slider, Thumb.END)).toBe(15);
+    });
+
+    it('should update the value on slide', async () => {
+      await slideToValue(slider, 35, Thumb.END);
+      expect(await getSliderValue(slider, Thumb.END)).toBe(35);
+    });
+  });
+
+  describe('disabled slider', async () => {
+    let slider: ElementFinder;
+    beforeAll(() => { slider = getDisabledSlider(); });
+
+    it('should not update the value on click', async () => {
+      await setValueByClick(slider, 15);
+      expect(await getSliderValue(slider, Thumb.END)).not.toBe(15);
+    });
+
+    it('should not update the value on slide', async () => {
+      await slideToValue(slider, 35, Thumb.END);
+      expect(await getSliderValue(slider, Thumb.END)).not.toBe(35);
+    });
+  });
+
+  describe('range slider', async () => {
+    let slider: ElementFinder;
+    beforeAll(() => { slider = getRangeSlider(); });
```

Comment:
Shouldn't we be calling this in `beforeEach`? The `beforeEach` at the top is set up to refresh the page before every test.

Reply:
I'm not sure, but if that's the case, shouldn't these tests fail?

Reply:
It might just be running the tests against the instance of the first element you got a hold of. Either way, switching it to a `beforeEach` won't slow things down..